app/controllers/backend/tax_declarations_controller.rb

```diff
@@ -64,7 +64,10 @@ def index
 
       respond_to do |format|
         format.html do
-          render "alert_no_VAT_declaration" unless FinancialYear.where.not(tax_declaration_mode: "none").any?
+          no_financial_year_opened = FinancialYear.opened.empty?
+          current_financial_year_with_tax_declaration = FinancialYear.with_tax_declaration.exclude?(FinancialYear.current)
```

Comment:
This implies that the variable equals `true` when the current financial year has a TaxDeclaration, but you're actually looking for the opposite. Is this properly named?